include/library/spdm_return_status.h

```diff
@@ -128,9 +128,13 @@ typedef uint32_t libspdm_return_t;
 #define LIBSPDM_STATUS_SESSION_MSG_ERROR \
     LIBSPDM_STATUS_CONSTRUCT(LIBSPDM_SEVERITY_ERROR, LIBSPDM_SOURCE_CORE, 0x000f)
 
+/* Re-triable decrypt error from peer - must rollback to backup keys. */
+#define LIBSPDM_STATUS_SESSION_MSG_ERROR_TRY_DISCARD_KEY_UPDATE \
+    LIBSPDM_STATUS_CONSTRUCT(LIBSPDM_SEVERITY_ERROR, LIBSPDM_SOURCE_CORE, 0x0010)
+
 /* Unable to acquire resource. */
 #define LIBSPDM_STATUS_ACQUIRE_FAIL \
-    LIBSPDM_STATUS_CONSTRUCT(LIBSPDM_SEVERITY_ERROR, LIBSPDM_SOURCE_CORE, 0x0010)
+    LIBSPDM_STATUS_CONSTRUCT(LIBSPDM_SEVERITY_ERROR, LIBSPDM_SOURCE_CORE, 0x0011)
```

Comment:
I dont think we can change this.

Please append new status definition after 0x0010.